tests/epyccel/recognised_functions/test_numpy_funcs.py

```diff
@@ -137,6 +137,52 @@ def fabs_phrase_r_i(x : 'int', y : 'float'):
     assert(isclose(f2(x,-y), fabs_phrase_r_i(x,-y), rtol=RTOL, atol=ATOL))
     assert(isclose(f2(-x,y), fabs_phrase_r_i(-x,y), rtol=RTOL, atol=ATOL))
 
+#------------------------------ isnan function ----------------------------#
+def test_numpy_isnan(language):
+    def numpy_isnan_test(x):
+        from numpy import isnan
+        return isnan(x)
```

Comment:
These functions are missing type annotations to allow Pyccel to deduce the types for the translation. See https://github.com/pyccel/pyccel/blob/devel/docs/quickstart.md#type-annotations
and https://github.com/pyccel/pyccel/blob/devel/docs/templates.md